tests/acceptance/cli/ursula/test_stakeholder_and_ursula.py

```diff
@@ -691,7 +691,7 @@ def test_collect_rewards_integration(click_runner,
 
     # Policy Fee
     collected_policy_fee = testerchain.client.get_balance(burner_wallet.address)
-    expected_collection = policy_rate * 30
+    expected_collection = policy_value
```

Comment:
This test has simply changed from collecting 30 period's worth to 1 period's worth – or is there something else to note here?

Reply:
`policy_rate * 30` confuses me a lot, because understanding is not accurate - it's not for 30 periods (only for 3), but test worked because `policy_value==policy_rate * 30`. `policy_value` is what we really had here